src/Illuminate/Mail/MailManager.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace Illuminate\Mail;
+
+use Illuminate\Support\Arr;
+use InvalidArgumentException;
+
+class MailManager
+{
+    /**
+     * The application instance.
+     *
+     * @var \Illuminate\Foundation\Application
+     */
+    protected $app;
+
+    /**
+     * The active connection instances.
+     *
+     * @var array
+     */
+    protected $connections = [];
+
+    /**
+     * The custom connection resolvers.
+     *
+     * @var array
+     */
+    protected $extensions = [];
+
+    /**
+     * Create a new mail manager instance.
+     *
+     * @param  \Illuminate\Foundation\Application  $app
+     * @param  \Illuminate\Mail\TransportFactory   $factory
+     * @return void
+     */
+    public function __construct($app, TransportFactory $factory)
+    {
+        $this->app = $app;
+        $this->factory = $factory;
+    }
+
+    /**
+     * Get a mailer connection instance.
+     *
+     * @param  string|null  $name
+     * @return \Illuminate\Mail\Mailer
+     */
+    public function connection($name = null)
```

Comment:
```PHP
public function connection(?string $name = null)
```

Reply:
The framework code doesn't make use of primitive type hints.